rest-api/tools/import_organizations.py

```diff
@@ -584,7 +584,7 @@ def main(args):
   HPODao()._invalidate_cache()
   OrganizationImporter().run(args.organization_file, args.dry_run)
   OrganizationDao()._invalidate_cache()
-  SiteImporter().run(args.site_file, args.dry_run)
+  SiteImporter().run(args.site_file, args.dry_run, args.creds_file)
```

Comment:
I don't think you should pass in creds_file anymore? Does this actually work?

Reply:
Nope, fixed.